### PR TITLE
Added note to refer users to sgkit

### PR DESCRIPTION
**Pull request by jeromekelleher**
As discussed offline with @alimanfoo, this is a note to point interested users towards sgkit. I don't think it's quite right at the moment, but here's a draft to get us started.

A couple of points:

- "Deprecation notice" seemed too strong, but I couldn't think of anything better
- Do we want to make concrete recommendations about who *should* use sgkit over sckit-allel?

cc @tomwhite @eric-czech @hammer

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -18,3 +18,16 @@ A Python package for exploratory analysis of large scale genetic variation data.
 
 .. image:: https://ci.appveyor.com/api/projects/status/1fna2yr9uht0u296?svg=true
     :target: https://ci.appveyor.com/project/alimanfoo/scikit-allel
+
+
+Deprecation notice
+******************
+
+Please note that scikit-allel is no longer under active development
+and will soon be in "maintenance-only" mode. The intended successor
```
Comment (alimanfoo) [inline README.rst:27]:
```suggestion
and is now in "maintenance-only" mode. The intended successor
```

File: README.rst [hunk 2]
```diff
@@ -18,3 +18,16 @@ A Python package for exploratory analysis of large scale genetic variation data.
 
 .. image:: https://ci.appveyor.com/api/projects/status/1fna2yr9uht0u296?svg=true
     :target: https://ci.appveyor.com/project/alimanfoo/scikit-allel
+
+
+Deprecation notice
```
Comment (alimanfoo) [inline README.rst:23]:
```suggestion
Development status
```

Perhaps "deprecation" is a little strong?

**Review by alimanfoo — COMMENTED** [review]
Hi @jeromekelleher, looks good, couple of small suggestions.

File: README.rst (hunk 2 above)
Comment (jeromekelleher) [reply README.rst:23]: Yep, SGTM

**Comment by jeromekelleher** [comment]
Thanks @alimanfoo, suggestions applied!

**Review by alimanfoo — APPROVED** [review]

**Comment by alimanfoo** [comment]
Great, thanks @jeromekelleher.

**Pull request merged**